clusterloader2/pkg/measurement/common/service_creation_latency.go

```diff
@@ -50,7 +50,11 @@ const (
 )
 
 func init() {
-	measurement.Register(serviceCreationLatencyName, createServiceCreationLatencyMeasurement)
+	err := measurement.Register(serviceCreationLatencyName, createServiceCreationLatencyMeasurement)
```

Comment:
Please look at this:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/measurement/common/etcd_metrics.go#L39

and make it consistent

Reply:
Same in other similar places.

Reply:
fixed for other similar places